lib/utilities.js

```diff
@@ -61,7 +62,52 @@ function invertObject(obj, override) {
   return result;
 }
 
+/**
+ * Check if a given tag is a custom component.
+ *
+ * @see https://github.com/facebook/react/blob/master/packages/react-dom/src/shared/isCustomComponent.js
+ * @param {string} tagName - The name of the html tag.
+ * @param {Object} props   - The props being passed to the element.
+ * @return {boolean}
+ */
+function isCustomComponent(tagName, props) {
+  if (tagName.indexOf('-') === -1) {
+    return props && typeof props.is === 'string';
```

Comment:
**docs:** do you mind adding a comment about the custom elements specification (https://www.w3.org/TR/2016/WD-custom-elements-20161013/#attr-is)